src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -118,11 +154,27 @@
                                       (reset! ready-to-lock? false))
                                     (and (not @locked?) (not @reviewing-audio?))
                                     (do
+                                      (audio/stop-recording
+                                       @recorder-ref
+                                       (cond
+                                         (and @ready-to-send? on-send)
+                                         #(do
```

Comment:
Avoid multiple forms when using the literal `#()`, especially in cases where there's no argument being used. I wish the linter could catch this... It probably can, but I haven't looked.

https://github.com/bbatsov/clojure-style-guide#no-multiple-forms-in-function-literals

Reply:
Done, I think I refactored all the cases in the PR